src/palace/manager/api/overdrive/constants.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from palace.manager.sqlalchemy.constants import DataSourceConstants
+
+
+class OverdriveConstants:
```

Comment:
I think it would make sense to move all these constants outside of this class at some point, but I was trying to only make the minimal necessary changes in this PR, so I left them in here for now.